backend/tests/v5/scripts/teamspaces/updateLicense.test.js

```diff
@@ -0,0 +1,147 @@
+/**
+ *  Copyright (C) 2022 3D Repo Ltd
+ *
+ *  This program is free software: you can redistribute it and/or modify
+ *  it under the terms of the GNU Affero General Public License as
+ *  published by the Free Software Foundation, either version 3 of the
+ *  License, or (at your option) any later version.
+ *
+ *  This program is distributed in the hope that it will be useful,
+ *  but WITHOUT ANY WARRANTY; without even the implied warranty of
+ *  MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ *  GNU Affero General Public License for more details.
+ *
+ *  You should have received a copy of the GNU Affero General Public License
+ *  along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+
+const {
+	determineTestGroup,
+	resetFileshare,
+	db: { reset: resetDB, createTeamspace },
+	generateRandomString,
+	generateRandomDate,
+} = require('../../helper/services');
+
+const { src, utilScripts } = require('../../helper/path');
+
+const UpdateLicense = require(`${utilScripts}/teamspaces/updateLicense`);
+
+const { editSubscriptions, getSubscriptions } = require(`${src}/models/teamspaces`);
+const { SUBSCRIPTION_TYPES } = require(`${src}/models/teamspaces.constants`);
+const { templates } = require(`${src}/utils/responseCodes`);
+
+const { isEmpty } = require(`${src}/utils/helper/objects`);
+
+const [ENTERPRISE,, DISCRETIONARY] = SUBSCRIPTION_TYPES;
```

Comment:
one comma can go

Reply:
That's intentional, i'm skipping `pilot` from the array 
![image](https://user-images.githubusercontent.com/11945337/211360768-27170318-6088-4093-b570-72bfcaf40284.png)

![image](https://user-images.githubusercontent.com/11945337/211360946-f3cd5089-717c-4fbf-873d-328d375f7ade.png)